app/src/main/java/org/esbtools/message/admin/rest/KeyResourceBean.java

```diff
@@ -134,10 +135,19 @@ public MetadataResponse delete(@PathParam("id") Long id) {
     @POST
     @Path("/sync/{entity}/{system}/{key}")
     @Produces({MediaType.APPLICATION_JSON})
-    public void sync(@PathParam("entity") String entity,
+    public MetadataResponse sync(@PathParam("entity") String entity,
                                @PathParam("system") String system,
                                @PathParam("key") String key,
                                @QueryParam("values") String values) {
+
+        if(!StringUtils.isBlank(entity) &&
```

Comment:
If we are checking that the parameter is not blank, we can use the isNotBlank method instead of using ! before, as it is slightly easier to read